terraform/india/production/variables.tf

```diff
@@ -40,6 +46,11 @@ variable "analysis_dashboard_version" {
     default = "main"
 }
 
+variable "sentry_dsn" {
+  type = string
+  description = "DNS for Sentry monitoring"
+}
```

Comment:
if you add a default of "" then it should be ok

Reply:
Thanks that seemed to do t he trick, wasn't sure if it needed a sensible default